32blit-stm32/STM32H750VBTx_FLASH.ld

```diff
@@ -33,7 +33,7 @@
 ENTRY(Reset_Handler)
 
 /* Highest address of the user mode stack */
-_estack = 0x20020000;    /* end of RAM */
+_estack = 0x2405A800;    /* end of RAM */
```

Comment:
Linker change...

Reply:
So the idea is to have .data and stack in DTCMRAM?

Reply:
I was mainly pointing out that whatever is decided, it probably doesn't belong here. (In a commit about MSC)...

Reply:
Ah ok.